src/components/DoughnutChart/DoughnutChart.tsx

```diff
@@ -0,0 +1,212 @@
+import React, { FC, useRef, useState } from "react";
+import classNames from "classnames";
+import Tooltip from "components/Tooltip";
+import "./DoughnutChart.scss";
+
+export type Segment = {
+  /**
+   * The colour of the segment.
+   */
+  color: string;
+  /**
+   * The segment tooltip.
+   */
+  tooltip?: string;
+  /**
+   * The segment length.
+   */
+  value: number;
+};
+
+export type Props = {
+  /**
+   * The label in the centre of the doughnut.
+   */
+  label?: string;
+  /**
+   * An optional class name applied to the wrapping element.
+   */
+  className?: string;
+  /**
+   * The width of the segments when hovered.
+   */
+  segmentHoverWidth: number;
+  /**
+   * The width of the segments.
+   */
+  segmentWidth: number;
+  /**
+   * The doughnut segments.
+   */
+  segments: Segment[];
+  /**
+   * The size of the doughnut.
+   */
+  size: number;
+  /**
+   *  ID associated to the specific instance of a Chart.
+   */
+  chartID: string;
+};
+
+export enum TestIds {
+  Label = "label",
+  Segment = "segment",
+  Chart = "chart",
+  Section = "Section",
+}
+
+const DoughnutChart: FC<Props> = ({
+  className,
+  label,
+  segmentHoverWidth,
+  segmentWidth,
+  segments,
+  size,
+  chartID,
+}): JSX.Element => {
+  const [tooltipMessage, setTooltipMessage] = useState<
+    Segment["tooltip"] | null
+  >(null);
+
+  const id = useRef(`doughnut-chart-${chartID}`);
+  const hoverIncrease = segmentHoverWidth - segmentWidth;
+  const adjustedHoverWidth = segmentHoverWidth + hoverIncrease;
+  // The canvas needs enough space so that the hover state does not get cut off.
+  const canvasSize = size + adjustedHoverWidth - segmentWidth;
+  const diameter = size - segmentWidth;
+  const radius = diameter / 2;
+  const circumference = Math.round(diameter * Math.PI);
+  // Calculate the total value of all segments.
+  const total = segments.reduce(
+    (totalValue, segment) => (totalValue += segment.value),
+    0,
+  );
+  let accumulatedLength = 0;
+  const segmentNodes = segments.map(({ color, tooltip, value }, i) => {
+    // The start position is the value of all previous segments.
+    const startPosition = accumulatedLength;
+    // The length of the segment (as a portion of the doughnut circumference)
+    const segmentLength = (value / total) * circumference;
+    // The space left until the end of the circle.
+    const remainingSpace = circumference - (segmentLength + startPosition);
+    // Add this segment length to the running tally.
+    accumulatedLength += segmentLength;
+
+    return (
+      <circle
+        className="doughnut-chart__segment"
+        cx={radius - segmentWidth / 2 - hoverIncrease}
+        cy={radius + segmentWidth / 2 + hoverIncrease}
+        data-testid={TestIds.Segment}
+        key={i}
```

Comment:
```suggestion
        key={i}
        tabIndex={0}
        aria-label={`${tooltip}: ${value}`}
```
If you do something like this, the segments will also be tab-selectable and announce their contents.

Reply:
It would also be nice to make the tooltip of a focused segment visible, so the tooltips appear on focus as well. Though that may be a bit more difficult and to be tackled as a separate issue.